scripts/deploy-helper.sh

```diff
@@ -40,7 +40,7 @@ function deploy_cluster_with_custom_image() {
 }
 
 function deploy_cluster() {
-  local operator_default="rook/ceph:master"
+  local operator_default="rook/ceph:v1.10.10"
```

Comment:
Is this the latest version that we know of that passes the tests ?

Reply:
This is the latest released version as of Jan 18th (and it passes the tests ;-))